starlark/src/values/error.rs

```diff
@@ -35,7 +35,6 @@ pub const INTERPOLATION_NOT_ENOUGH_PARAMS_ERROR_CODE: &str = "CV10";
 pub const INTERPOLATION_VALUE_IS_NOT_CHAR_ERROR_CODE: &str = "CV12";
 pub const TOO_MANY_RECURSION_LEVEL_ERROR_CODE: &str = "CV13";
 pub const UNSUPPORTED_RECURSIVE_DATA_STRUCTURE_ERROR_CODE: &str = "CV14";
-pub const CANNOT_MUTATE_DURING_ITERATION_ERROR_CODE: &str = "CV15";
```

Comment:
Use some kind of reservation so we don't reallocate the error code in the future, e.g.:
```
const __RESERVED_CV15: &str = "CV15"
```